misc/build_helpers/build-manylinux-wheels.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+# This runs in the container to actually build the wheels.
+BUILDER="/io/misc/build_helpers/_build-wheels.sh"
+
+# This image can build x86_64 (64 bit) manylinux wheels.
+DOCKER_IMAGE="quay.io/pypa/manylinux1_x86_64"
```

Comment:
I'll see if there's a way we can use Docker's image signature feature ... which may or may not help with confidence we're running what we want to run.